README.md

```diff
@@ -28,6 +28,22 @@ The `kerberos` package is a C++ extension for Node.js that provides cross-platfo
   2. Install [Python 2.7](https://www.python.org/downloads/) or [Miniconda 2.7](http://conda.pydata.org/miniconda.html) (`v3.x.x` is not supported), and run `npm config set python python2.7`
   3. Launch cmd, `npm config set msvs_version 2015`
 
+**z/OS**
+- Install [IBM SDK for node.js v12.16.1 or later](https://www.ibm.com/ca-en/marketplace/sdk-nodejs-compiler-zos/purchase).
+- Install krb5 libraries for z/OS (which requires [curl for z/OS](https://www.rocketsoftware.com/product-categories/mainframe/curl-for-zos) and [gzip for z/OS](https://www.rocketsoftware.com/product-categories/mainframe/gzip-zos))
```

Comment:
Suggest to end the 2nd Install statement with a ':' - so it's clear that the next commands are for installing krb5..